tests/src/test/scala/it/unibo/scafi/test/functional/stdlib/FieldUtils.scala

```diff
@@ -0,0 +1,205 @@
+package it.unibo.scafi.test.functional.stdlib
+
+import it.unibo.scafi.test.FunctionalTestIncarnation._
+import it.unibo.scafi.test.functional.{ScafiAssertions, ScafiTestUtils}
+import org.scalatest._
+
+
+class FieldUtils extends FlatSpec {
+  import ScafiAssertions._
+  import ScafiTestUtils._
+
+  val Field_Utils = new ItWord
+
+
+  private[this] trait SimulationContextFixture {
+    val net: Network with SimulatorOps = standardNetwork()
+  }
+
+  private[this] trait TestProgram extends AggregateProgram with StandardSensors with BlockG
+
+  Field_Utils should "support min/maxHoodSelectors" in new SimulationContextFixture {
+    // ACT
+    exec(new TestProgram {
+      override def main(): Any = (
+        excludingSelf.minHoodSelector(nbr(mid()))(nbr(mid())),
+        includingSelf.minHoodSelector(nbr(mid()))(nbr(mid())),
+        excludingSelf.maxHoodSelector(nbr(mid()))(nbr(mid())),
+        includingSelf.maxHoodSelector(nbr(mid()))(nbr(mid()))
+      )
+    }, ntimes = fewRounds)(net)
+
+    // ASSERT
+    assertNetworkValues((0 to 8).zip(List(
+      (Some(1),0,Some(4),4), (Some(0),0,Some(5),5), (Some(1),1,Some(5),5),
+      (Some(0),0,Some(7),7), (Some(0),0,Some(7),7), (Some(1),1,Some(7),7),
+      (Some(3),3,Some(7),7), (Some(3),3,Some(6),7), (None,8,None,8)
+    )).toMap)(net)
+
+  }
+
+  Field_Utils should "support anyHood" in new SimulationContextFixture {
+    // ACT
+    exec(new TestProgram {
+      override def main(): Any = (
+        excludingSelf.anyHood(mid() == nbr(mid())),
+        includingSelf.anyHood(mid() == nbr(mid()))
+      )
+    }, ntimes = fewRounds)(net)
+
+    //ASSERT
+    assertNetworkValues((0 to 8).zip(List(
+      (false, true), (false, true), (false, true),
+      (false, true), (false, true), (false, true),
+      (false, true), (false, true), (false, true)
+    )).toMap)(net)
+  }
+
+  Field_Utils should "support anyHood, one neighbour with true value" in new SimulationContextFixture {
```

Comment:
As I said, please try to cover all cases within a single scenario.
That is, build a single network where there are:

- 1 or more devices with zero neighbours with value `true`
- 1 or more devices with just one neighbour with value `true`
- 1 or more devices with more than one neighbours with value `true`
- 1 detached device with `true` value
- 1 detached device with `false` value

E.g., an input network like:

```
F    T  F
T    F  [F]
[T]  F  F
```
(where `[x]` denotes a detached device with no neighbours with value `x`) (you may also use a 4x4 net if you find it more appropriate)

That is, you should merge this test case with the following one.